packages/fast-components-react-base/src/select/select.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Select",
+    "description": "A select component's schema definition.",
+    "type": "object",
+    "id": "select",
+    "properties": {
+        "placeholder": {
+            "title": "Placeholder",
+            "type": "string"
+        },
+        "name": {
+            "title": "Name",
+            "type": "string"
+        },
+        "form": {
+            "title": "Form",
```

Comment:
Form... ID?

Reply:
Sure, that makes for a more descriptive title.